### PR TITLE
Added the ability to select time within a certain range.

### PR DESCRIPTION
**Pull request by Adelxoxo**
Here is a video example: https://streamable.com/qdkum

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-time-picker/93)

### CODE REVIEW

**Comment by Adelxoxo** [comment]
From this issue https://github.com/vaadin/vaadin-time-picker/issues/47

File: src/vaadin-time-picker.html [hunk 1]
```diff
@@ -87,7 +89,7 @@
        * ----------------|----------------
        * `toggle-button` | The toggle button
        *
-       * See [ThemableMixin – how to apply styles for shadow parts](https://github.com/vaadin/vaadin-themable-mixin/wiki)
+       * See [ThemableMixin â€“ how to apply styles for shadow parts](https://github.com/vaadin/vaadin-themable-mixin/wiki)
```
Comment (limonte) [inline src/vaadin-time-picker.html]: something weird has happened here:

image

**Comment by Adelxoxo** [comment]
@limonte Okay, I pushed a commit with a test now. I'm not used to writing these so I took example on the step test, but I could still be missing something obvious as this type of test is not that common in my industry.

File: src/vaadin-time-picker.html (hunk 1 above)
Comment (Adelxoxo) [reply src/vaadin-time-picker.html]: Fixed, see below.

**Comment by limonte** [comment]
Good day @Adelxoxo 

We have a unique situation here, two PRs adding the same functionality - yours (#93) and #92 

#92 was added prior to yours and we like the implementation more. It's incomplete though, and I asked its author to add some tests which I took from your PR.

Most probably #92 will be merged and released, but it's not 100% sure for now. 

Anyways, thank you for this PR, I'll keep you updated in this thread.

**Comment by Adelxoxo** [comment]
Hi @limonte,

It's your decision, but personally I don't think the date it was added should be a factor in this decision as long as as both PR come within the same time frame, it would set a bad precedent for yourself where pushing for incomplete features in order to get priority on bounties becomes a thing. I myself could have made a PR with incomplete features earlier but I decided to finish working on the features the next day before doing so. However I would appreciate to know what you didn't like about my implementation, I'm thinking about investing myself more into open source projects and I would appreciate any feedback.

**Pull request closed without merging**